index.js

```diff
@@ -222,6 +223,29 @@ var putFile = function (bucket, key, filepath) {
     });
 };
 
+/* Put an object in S3
+ *
+ * Parameters:
+ *  any valid parameters listed that aws-sdk.s3().upload() supports
+ *  https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
+ *
+ * Return:
+ *    bluebird promise
+ */
+var upload = function (params) {
+    return new BluebirdPromise(function (resolve, reject) {
```

Comment:
Total nitpick: indentation on 236 and 246. Fix if you want to, no pressure at all.

Reply:
Ya I do want to dig in some more and see what the consequences of this lack expiration are!
Frustratingly, I couldn't find in the official AWS SDK docs what the default expiration time is!  I did find on some random blog that it's only 15 minutes...and that seems to be right base on my testing!

It looks like we were mainly using this method when requesting image URLs... I'd have to dig more to see in what context in the app that we're using these image URL's.